packages/riipen-ui/src/components/Container.test.jsx

```diff
@@ -0,0 +1,104 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import defaultTheme from "../themes/default";
+
+import Container from "./Container";
+
+describe("<Container>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Container />);
+    } catch (theError) {
+      error = theError;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Container().type.defaultProps;
+
+      const wrapper = mount(<Container />);
+
+      expect(wrapper.find("Container").props().maxWidth).toEqual(
+        defaultProps.maxWidth
+      );
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("appends higher order values to default classes prop with withClass decorator", () => {
+      const wrapper = mount(<Container />);
+
+      expect(wrapper.find("Container").props().classes).toEqual([
+        "riipen",
+        "riipen-container"
+      ]);
+    });
```

Comment:
@imogenem and you are writing these tests in slightly different places. Imo is putting this inside the `describe('classes prop'` and you are putting them in the `default props` block. We should be consistent. 

My two cents is probably to prefer it in the `classes` block since it's actually concatenating the HOC's classes to the component.

Reply:
Moved this test to the classes prop describe block.